bolt-modules/boltlib/lib/puppet/functions/set_var.rb

```diff
@@ -0,0 +1,31 @@
+# Sets a variable { key => value } for a target
+#
+# This function takes 3 parameters:
+# * A Target object to set the variable for
+# * The key for the variable (String)
+# * The value of the variable (Any)
```

Comment:
We may expand this to `Any` eventually, but it should match the argument type `Data`.